metadata-app/src/api/index.js

```diff
@@ -20,6 +20,32 @@ const decodeExtras = (opts) => {
   return newOpts;
 };
 
+/**
+ * Iterates over each field and if it's string encodes special
+ * characters
+ * @param {Object} obj
+ */
+const encodeValuesURIComponent = (obj) => {
```

Comment:
@kmanaseryan is it needed for IE11?

Reply:
No, this for the other issue # 97

Reply:
@kmanaseryan this doesn't look right... why only encode the fields? Isn't it possible for the keys to include `&` in keys as well?

I was thinking we would need to stringify and encode the entire payload. Did you see differently?

Reply:
OK, I see. I just didn't think about the field names, because intuitively I thought only the field values, becuase these are the things that can be anything, in contrast to the field names, which are defined and we know, but I see what you are saying. So I guess since we can have any field name later in the future, then that would be better if we encode the entire payload.